src/reducers/listReducer.js

```diff
@@ -95,6 +95,15 @@ export default (name = '', transformer = null, comparator = null) => {
       }
     }
 
+    if (result.removeAll) {
+      return {
+        ...state,
+        sequence: ++state.sequence,
+        list: [],
+        transformedList: state.transformedList
```

Comment:
`transformedList` is essentially

```
list.map(this.transform)
```
so if you change `list`, you have to change `transformedList`

Reply:
I am unclear how transformedList works. None of the listReducers that are created in the uiReducer pass in a transformer. My understanding of the transformedList is after the list is updated, a transformed list can be computed based on a transform function that is passed into the listReducer. Since removeAll is being used to clear all the contents of a list shouldn't the transformedList also be an empty list similar to list?